src/pybind/mgr/dashboard/services/access_control.py

```diff
@@ -460,6 +461,17 @@ def create_user(self, username, password, name, email, enabled=True,
             self.users[username] = user
             return user
 
+    def save_attempt(self, username, attempt):
+        with self.lock:
+            if username in self.users:
+                self.attempt[username] = attempt
+
+    def get_attempt(self, username):
+        with self.lock:
+            if username in self.attempt.keys():
+                return self.attempt[username]
+            return 0
```

Comment:
This is the [Pythonic way](https://stackoverflow.com/questions/12265451/ask-forgiveness-not-permission-explain):
```suggestion
            try:
                return self.attempt[username]
            except KeyError:
                return 0
```

Reply:
Thank you. Still getting used to the pythonic way. But seeing this there are lot of areas in our code this method can be adapted right?

Reply:
Yup, but I don't it's worthy such a huge clean-up. Let's just do this for new and refactored code.

Reply:
Acknowledged.